low-level/files/opt/seagate/sspl/conf/sspl.conf.LDR_R1

```diff
@@ -101,6 +101,7 @@ IP_addr=172.16.1.222
 max_login_attempts=2
 
 [RAIDSENSOR]
+should_monitor=XXXXXX # RAIDSENSOR
```

Comment:
Why "XXXXXX"? can we put true by default?

Reply:
Same comment for others below.

Reply:
In progress.

Reply:
Fixed